### DISCUSSION TITLE
Redis no longer open-source what do you suggest as an alternative backend?

### DISCUSSION THREAD

**Discussion by IvoLeist** [celery/celery#8951] [General]
Hi Celery community,

since Redis is no longer open-source to what backend are you pivoting and why?

@ Celery Devs is there any plans yet to support [Valkey](https://github.com/valkey-io/valkey) in the (near) future?

**Comment by Nusnus** [comment]
> since Redis is no longer open-source

Can you please share more context?

**Reply by IvoLeist** [reply]
https://www.infoq.com/news/2024/03/redis-license-open-source/
